src/algokit/core/tasks/vanity_address.py

```diff
@@ -124,7 +124,7 @@ def generate_vanity_address(keyword: str, match: MatchType) -> VanityAccount:
     """
     jobs: list[Process] = []
 
-    def signal_handler(sig: int, frame: types.FrameType | None) -> Any | None:
+    def signal_handler(sig: int, frame: types.FrameType | None) -> Any | None:  # noqa: ANN401
```

Comment:
you could type this as returning `object` instead I think? then no need for the `noqa`